test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -396,6 +396,214 @@ def check_lpd(prototypes_base)
     end
   end
 
+  def check_hvac_sizing(prototypes_base)
```

Comment:
We probably want to add the YARD comments before merging.

Reply:
done